lib/bundler/plugin.rb

```diff
@@ -197,7 +197,16 @@ def load_plugin(name)
       # done to avoid conflicts
       path = index.plugin_path(name)
 
+      if insert_index = Bundler.rubygems.load_path_insert_index
```

Comment:
Yes, I am working on extending the serializer to with array. As soon as I get around that, will fix this.